examples/ICESat-2_DAAC_DataAccess_Example.ipynb

```diff
@@ -184,26 +178,27 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "region_a = ipx.Query(short_name, spatial_extent, date_range, \\\n",
-    "   start_time='03:30:00', end_time='21:30:00', version='002')\n",
-    "\n",
     "print(region_a.dataset)\n",
     "print(region_a.dates)\n",
     "print(region_a.start_time)\n",
     "print(region_a.end_time)\n",
-    "print(region_a.dataset_version)\n",
     "print(region_a.cycles)\n",
     "print(region_a.tracks)\n",
-    "# print(region_a.spatial_extent)"
+    "print(region_a.dataset_version)\n",
+    "# print(region_a.spatial_extent)\n",
+    "region_a.visualize_spatial_extent()"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Finally, there are also two optional inputs to allow the user to search based on orbital parameters. Note that your search will return a no-data error if your cycles or tracks are not withint your spatial extent.\n",
-    "- `cycles` = Which orbital cycle of the dataset to use, input as a numerical string or a list of strings. If no input is given, this value defaults to all available cycles within the search parameters.  An orbital cycle refers to the 91-day repeat period of the ICESat-2 orbit.\n",
-    "- `tracks` = Which [Reference Ground Track (RGT)](https://icesat-2.gsfc.nasa.gov/science/specs) of the dataset to use, input as a numerical string or a list of strings. If no input is given, this value defaults to all available RGTs within the spatial and temporal search parameters."
+    "There are also several optional inputs to allow the user finer control over their search. Start and end time are only valid inputs on a temporally limited search.\n",
+    "- `start_time` = start time to search for data on the start date. If no input is given, this defaults to 00:00:00.\n",
+    "- `end_time` = end time for the end date of the temporal search parameter. If no input is given, this defaults to 23:59:59. Times must be input as 'HH:mm:ss' strings.\n",
+    "- `version` = What version of the dataset to use, input as a numerical string. If no input is given, this value defaults to the most recent version of the dataset specified in `short_name`.\n",
+    "\n",
+    "*NOTE* Version 001 is used as an example in the below cell. However, using it will cause 'no results' errors in granule ordering for some search parameters. These issues have been resolved in later versions of the datasets, so it is best to use the most recent version where possible. Thus, you will need to change the version associated with `region_a` and rerun the next cell for the rest of this notebook to run."
```

Comment:
Looks like ATL06 version 002 is used below.

```suggestion
    "*NOTE* Version 002 is used as an example in the below cell. However, using it will cause 'no results' errors in granule ordering for some search parameters. These issues have been resolved in later versions of the datasets, so it is best to use the most recent version where possible. Thus, you will need to change the version associated with `region_a` and rerun the next cell for the rest of this notebook to run."
```

Reply:
Haha - nice catch. I just spotted this this morning and added it to my to-fix list!